### PR TITLE
introduced prop for typeahead to extend upwards

### PR DESCRIPTION
**Pull request by hasznosl**
Checklist:

- [x] Test added / Snapshots updated
- [ ] Documentation added / updated

### CODE REVIEW

File: stories/TypeaheadStory.jsx [hunk 1]
```diff
@@ -134,6 +134,7 @@ class TypeaheadStory extends React.Component {
               onInputValueChange={this.onInputChange}
               items={movies}
               isLoading={loading}
+              extendUpwards={true}
```
Comment (yamafaktory) [inline stories/TypeaheadStory.jsx]: We should have another scenario in the story instead of changing that one.

**Comment by yamafaktory** [comment]
@hasznosl Can you update the snaps please 🖼 📸 ?

File: stories/TypeaheadStory.jsx (hunk 1 above)
Comment (hasznosl) [reply stories/TypeaheadStory.jsx]: thats at the bottom anyway, but yea I can do that

File: src/organisms/Typeahead.jsx [hunk 2]
```diff
@@ -67,6 +82,7 @@ export default class Typeahead extends React.PureComponent {
     onClearSelection: NOOP,
     onClose: NOOP,
     onOpen: NOOP,
+    placement: 'bottom',
```
Comment (dan-lee) [inline src/organisms/Typeahead.jsx]:
```suggestion
    placement: Placement.bottom,
```

File: src/organisms/Typeahead.jsx [hunk 3]
```diff
@@ -31,6 +44,8 @@ export default class Typeahead extends React.PureComponent {
     clearOnSelect: PropTypes.bool,
     /** The default value of the component, without making it controlled. */
     defaultValue: PropTypes.string,
+    /** If "top", then the list should be reversed and extended upwards, if "bottom" (default) then downwards */
+    placement: PropTypes.string,
```
Comment (dan-lee) [inline src/organisms/Typeahead.jsx]:
```suggestion
    placement: PropTypes.oneOf(Object.values(Placement)),
```

**Pull request merged**